icu4c/source/i18n/unicode/basictz.h

```diff
@@ -152,6 +152,17 @@ class U_I18N_API BasicTimeZone: public TimeZone {
     virtual void getSimpleRulesNear(UDate date, InitialTimeZoneRule*& initial,
         AnnualTimeZoneRule*& std, AnnualTimeZoneRule*& dst, UErrorCode& status) const;
 
+#ifndef U_HIDE_DRAFT_API
```

Comment:
```suggestion
#ifndef U_FORCE_HIDE_DRAFT_API
```

Let's use the "FORCE" guards so that hiding these functions does not change the class vtables.

Please do so in all subclasses, too.

Reply:
done